config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -27,23 +29,46 @@ presubmits:
         resources:
           requests:
             cpu: 4
+  # this replaces the bootstrap / scenario based job going forward
+  - name: pull-kubernetes-integration
+    always_run: true
+    decorate: true
+    skip_branches:
+    - release-\d+.\d+ # per-release job
     annotations:
+      fork-per-release: "true"
```

Comment:
need an ack if this job's skip_branches config works with a fork-per-release approach

Reply:
It does. It gets replaced with a `branches` statement with the specific branch it's forking to.